packages/@coorpacademy-components/src/template/app-player/player/slides/style.css

```
@@ -105,16 +107,34 @@
 .guideWrapper {
   composes: infos;
   left: 0;
-  padding: 4px 10px;
-  border-bottom-right-radius: 12px;
   cursor: pointer;
-  border-top-left-radius: 3px;
+  border-radius: 3px;
+  margin-top: 24px;
+  margin-left: 24px;
+  padding: 12px;
 }
 
-.guideWrapper span {
+.guideWrapper[data-name="newMedia"] {
+  color: orangeAdd;
+  background-color: rgba(255, 112, 67, 0.1);
+}
+
+.guideWrapper[data-name="newMedia"] span {
   animation: fade 2.5s infinite linear;
 }
 
+.guideWrapper:hover {
+  color: white;
+  transition: 1s; 
+}
+
+.guideWrapper[data-name="newMedia"]:hover {
+  background-color: rgba(255, 112, 67, 1);
+}
+
+.guideWrapper[data-name="reviewMedia"]:hover {
+  background-color: rgba(96, 125, 139, 1);
```

```suggestion
  background-color: grey;
```
`rgba(96, 125, 139, 1) = #607D8B` => `grey`